hi-ml-azure/src/health_azure/utils.py

```diff
@@ -1459,6 +1459,34 @@ def download_files_from_run_id(
     torch_barrier()
 
 
+def get_driver_log_file_text(run: Run) -> Optional[str]:
+    """Given a run object, return the text stored in the driver log file of that run. Detects log files produced
+    by both old and new runtimes.
+
+    :param run: Run object representing the current run.
+    :return: Driver log file text if a file exists, None otherwise.
+    """
+    with tempfile.TemporaryDirectory() as tmp_dir_name:
+        run.download_files(
```

Comment:
I would not download the whole azureml-logs/user_logs folders, they also contain potentially large files, restrict to only the single file we need.

Reply:
or check the render_and_run_test_script function for an alternative way of seeing which files are available

Reply:
The second occurrence of `70_driver_log.txt` is in `docs/source/examples/2/results.py` which imports `health_azure` as a PyPI package - it will needs to be changed in a separate PR after this one is released in a new PyPI package.